rules.md

```diff
@@ -485,23 +396,23 @@ In other words, Tasks should be priced via a process to discover what is the min
 
 Fiat prices should be taken into consideration when a Task is created, but Tasks shouldn't be repriced if the fiat value changes unless not doing so would prevent completion of the Task.
```

Comment:
We actually practiced the opposite... adjusting prices in response to dramatic swings in the fiat pricing of Dash